doc/reference/ufunc.rst

```diff
@@ -5,7 +5,49 @@ Universal Functions (ufunc)
 
 .. https://docs.scipy.org/doc/numpy/reference/ufuncs.html
 
-DPNP provides universal functions (a.k.a. ufuncs) to support various element-wise operations.
+A universal function (or ufunc for short) is a function that operates on ndarrays in an element-by-element fashion, \
```

Comment:
Why do we need `\` here? Is it helps somehow to format the output? (and in what way?)
I don't see that we used it previously somewhere and so wonder why do we need it here.